### ISSUE TITLE
Switch to sleep_lights if sleep_entity turns on

### ISSUE THREAD

**Issue by sti0** [jseidl/hass-magic_areas#31]
I think that is a bug instead of a feature request. If the `sleep_entity` turns to on, the automatic lights should turn off and the `sleep_lights` should turn on. At the moment the area needs to be fully cleared and after the next presence the `sleep_lights` turn on.

In case of the `disable_entity` this behavior works well (`disable_entity` turns on, lights turn off immediately).

**Comment by jseidl** [comment]
Good catch. `disable_entity` already follows that premise, reasonable this should be on `sleep_entity` as well.

**Comment by jseidl** [comment]
This is being fixed on #126 for release 3.0.0
